### DISCUSSION TITLE
Unable to use the MVC 6 template

### DISCUSSION THREAD

**Discussion by pendraggon87** [Dotnet-Boxed/Templates#75] [Q&A]
Error message: The template specified cannot be found.

**Comment by RehanSaeed** [comment]
Have you got any more details than that? Which version of Visual Studio were you using. When do you get this error? A screenshot would be most helpful. From the few words above I can speculate that maybe your extension got corrupted? Try re-installing it maybe?

**Comment by RehanSaeed** [comment]
Reopen if you have more details.
